core/widgets/SubmitButtonWidget.php

```diff
@@ -42,9 +42,31 @@ class SubmitButtonWidget extends Widget
     public $options = [];
 
     /**
+     * @since 1.0.24 First time this was introduced
+     *
      * @var activeForm Define activeForm context to use widget in context and only disable button when validation succeeded
      */
-    public $activeForm;
+    protected $activeForm;
+
+    /**
+     * @since 1.0.24 First time this was introduced
+     *
+     * @param $activeForm Set $activeForm and check for type "ActiveForm"
+     */
+    public function setActiveForm($activeForm) {
+        if ($activeForm instanceof ActiveForm) {
+            $this->activeForm = $activeForm;
+        }
+    }
+
+    /**
+     * @since 1.0.24 First time this was introduced
+     *
+     * @return ActiveForm Return $activeForm
+     */
+    public function getActiveForm() {
```

Comment:
psr2 coding style:

public function getActiveForm()
{
    return $this->_activeForm;
}

Reply:
Please also take a look at how to use phpdocs from comments above